redbot/cogs/alias/alias_entry.py

```diff
@@ -30,34 +47,195 @@ def inc(self):
         self.uses += 1
         return self.uses
 
+    def get_extra_args_from_alias(self, message: discord.Message, prefix: str) -> str:
+        """
+        When an alias is executed by a user in chat this function tries
+            to get any extra arguments passed in with the call.
+            Whitespace will be trimmed from both ends.
+        :param message:
+        :param prefix:
+        :param alias:
+        :return:
+        """
+        known_content_length = len(prefix) + len(self.name)
+        extra = message.content[known_content_length:]
+        view = StringView(extra)
+        view.skip_ws()
+        extra = []
+        while not view.eof:
+            prev = view.index
+            word = view.get_quoted_word()
+            if len(word) < view.index - prev:
+                word = "".join((view.buffer[prev], word, view.buffer[view.index - 1]))
+            extra.append(word)
+            view.skip_ws()
+        return extra
+
     def to_json(self) -> dict:
-        try:
-            creator = str(self.creator.id)
-            guild = str(self.guild.id)
-        except AttributeError:
-            creator = self.creator
-            guild = self.guild
```

Comment:
This was removed because I ensured all creation of AliasEntry objects only accepted the ID not the object itself. I'll further ensure that with typehints to only accept int and not the object themselves. They served no purpose being the object and just a reference ID is good enough.

Reply:
I was basing this comment on `from_json` method which could change it into Member/Guild objects if bot was passed. I know it's never passed so maybe you could just remove this all together or at least add a comment that some changes to `AliasEntry` would be needed if we would start to pass the bot object